bin/gsnap_mapping.pl

```diff
@@ -153,7 +150,7 @@
 
 $output_format="sam";
 if ($output_format =~ /bam/i) {
-	#my $cmd2 = "$hadoop jar $sartsam_jar I=$sam_output O=$bam_output SO=coordinate VALIDATION_STRINGENCY=SILENT";
+	## TODO: WARNING: this won't work anymore and will fail!!!
 	my $cmd2 = "java -Xmx5g -Dsnappy.disable=true -jar $sartsam_jar I=$sam_output O=$bam_output SO=coordinate VALIDATION_STRINGENCY=SILENT";
```

Comment:
Here picard is indeed used

Reply:
@robmaz - do we really need this?

Reply:
I realized that this codepath is never tested, because the `$output_format="sam";` - removing all this code to proceed.